Description: Disable references to unpackaged maven plugins in lib module
Author: Miguel Landaeta <nomadium@debian.org>
Forwarded: no
Last-Update: 2017-03-08

--- a/lib/pom.xml
+++ b/lib/pom.xml
@@ -309,6 +309,7 @@ DO NOT MODIFIY - GENERATED CODE
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>build-helper-maven-plugin</artifactId>
       </plugin>
+      <!--
       <plugin>
         <groupId>io.takari.polyglot</groupId>
         <artifactId>polyglot-maven-plugin</artifactId>
@@ -367,6 +368,7 @@ DO NOT MODIFIY - GENERATED CODE
           </dependency>
         </dependencies>
       </plugin>
+      -->
     </plugins>
   </build>
 </project>
